lib/game.rb

```diff
@@ -0,0 +1,19 @@
+class Game
```

Comment:
Good job on making the code more readable by making the hash into an array of objects

Reply:
Agreed - It's much more readable this way 👍